storage/database/dynamodb.go

```diff
@@ -97,20 +101,27 @@ func createTestDynamoDBConfig() *DynamoDBConfig {
 		Region:             "ap-northeast-2",
 		Endpoint:           "https://dynamodb.ap-northeast-2.amazonaws.com",
 		TableName:          "dynamo-test-tmp",
-		ReadCapacityUnits:  dynamoReadCapacityUnits,
-		WriteCapacityUnits: dynamoWriteCapacityUnits,
+		IsProvisioned:      false,
+		ReadCapacityUnits:  10000,
+		WriteCapacityUnits: 10000,
 	}
 }
 
 func NewDynamoDB(config *DynamoDBConfig) (*dynamoDB, error) {
 	if config == nil {
 		return nil, nilDynamoConfigErr
 	}
+	if len(config.Endpoint) == 0 {
+		config.Endpoint = "https://dynamodb." + config.Region + ".amazonaws.com"
+	}
+	if len(config.TableName) == 0 {
+		config.TableName = "klaytn" + strconv.Itoa(time.Now().Nanosecond())
+	}
```

Comment:
How about terminating a node when the table name is not specified?
Since using S3 consumes users' AWS credits (or money), we should be careful.

Reply:
review reflected :)